demisto_sdk/commands/common/hook_validations/graph_validator.py

```diff
@@ -218,7 +218,10 @@ def is_file_using_unknown_content(self):
                 content_item.name, unknown_content_names
             )
             if self.handle_error(
-                error_message, error_code, content_item.path, warning=True
+                error_message,
+                error_code,
+                content_item.path,
+                warning=bool(self.file_paths),
```

Comment:
why `warning=bool(self.file_paths)`? please explain in a comment so future maintainers understand